RC: Maintenance windows | update docs (#3194)
RedisLabs/redislabs-docs: content/rc/subscriptions/maintenance/_index.md

> Redis will maintain your Redis Cloud subscriptions and databases as needed to ensure your databases are running the most stable and up-to-date version of Redis. During maintenance, you may notice some latency when connecting to your databases. 

Author: Fixed in [1bbee48](https://github.com/RedisLabs/redislabs-docs/pull/3194/commits/1bbee482e5a922619d09fe502aaa46a4107c6170).

- Adding more memory to a node
- Applying security patches

Redis will notify users by email when maintenance starts and ends. For more details, see [Notifications](#notifications).

> During maintenance, your application may disconnect from our service for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries.

Reviewer: Besides my previous comment about adding the point about the latency, I suggest starting the section with "during a maintenance activity, your database will be operational, but...

Author: Fixed in [1bbee48](https://github.com/RedisLabs/redislabs-docs/pull/3194/commits/1bbee482e5a922619d09fe502aaa46a4107c6170).

> During maintenance, your database will be operational, but you may notice some latency when connecting to your databases. Your application may also disconnect from your database for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries.

Reviewer: @cmilesb, maybe we should bold it in some way and not just embed it in the paragraph

Author: @NoamSternRedis Bolding or italicizing entire sections of text doesn't fit our style. I separated the "disconnection" part from the "latency" part into two separate paragraphs. Do you think that's enough, or should I use a note to bring attention to the disconnection? Or, is there a specific part of this section you think needs to be bolded?

### Urgent maintenance

Urgent maintenance refers to any activity that could affect service and cannot wait for scheduling. This includes applying urgent security patches.

> Redis can perform urgent maintenance at any time, even if you have set a manual maintenance window or have temporarily skipped maintenance. Redis will notify users by email when urgent maintenance starts and ends.

Reviewer: hyperlink on the words:  "temporarily skipped maintenance" to the relevant section - https://docs.redis.com/staging/DOC-3496/rc/subscriptions/maintenance/set-maintenance-windows/#skip-maintenance-temporarily

- Adding more memory to a node
- Applying security patches

Redis will notify users by email when maintenance starts and ends. For more details, see [Notifications](#notifications).

During maintenance, your database will be operational, but you may notice some latency when connecting to your databases. 

Your application may also disconnect from your database for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries.

### Urgent maintenance

Urgent maintenance refers to any activity that could affect service and cannot wait for scheduling. This includes applying urgent security patches.

> Redis can perform urgent maintenance at any time, even if you have set a manual maintenance window or have temporarily [skipped maintenance]({{<relref "/rc/subscriptions/maintenance/set-maintenance-windows#skip-maintenance-temporarily">}}). Redis will notify users by email when urgent maintenance starts and ends.

Reviewer 2: ```suggestion
Redis can perform urgent maintenance at any time, even if you have set a manual maintenance window or have temporarily [skipped maintenance]({{<relref "/rc/subscriptions/maintenance/set-maintenance-windows#skip-maintenance-temporarily">}}). Redis will notify you by email when urgent maintenance starts and ends.
```

## Notifications

> Redis will notify users by email when maintenance starts and ends. If Redis needs an action from a user to start maintenance, Redis will notify users with a reasonable amount of time before planned maintenance.

Reviewer 2: ```suggestion
Redis will notify you by email when maintenance starts and ends. If Redis needs an action from you to start maintenance, Redis will notify you with a reasonable amount of time before planned maintenance.
```

> For major upgrades or upgrades that might include breaking changes, users will receive an advance notification with sufficient time to prepare before the upgrade.

Reviewer 2: ```suggestion
For major upgrades or upgrades that might include breaking changes, you will receive an advance notification with sufficient time to prepare before the upgrade.
```
